.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,141 @@
+name: 🐛 Report a problem
+description: Report an issue with MagicMirror² 🚨
+title: "[Bug] {{ brief description }}"
+labels:
+  - bug
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for reporting a bug! Please fill in the following template to help us reproduce the issue.
+        Please only submit reproducible issues. If you're not sure if it's a real bug or if it's just you, please open a topic on the forum.
+  - type: textarea
+    id: environment
+    attributes:
+      label: Environment
+      description: |
+        Please tell us about how your MagicMirror² is set up.
+      value: |
+        MagicMirror² version:
+          (It can be found in the `package.json` file. Please use the latest version before reporting a bug.)
+        Node version: 
+          (Run `node -v` to find out. Make sure it's version 20 or later (recommended is 22))
+        npm version: 
+          (Run `npm -v` to find out.)
+        Platform: 
+          (Raspberry Pi 2/3/4/5, Windows, Mac, Linux, System V UNIX, ...)
+    validations:
+      required: true
+  - type: dropdown
+    id: start-option
+    attributes:
+      label: Which start option are you using?
+      description: |
+        Please keep in mind that some problems are specific to certain start options.
+      options:
+        - "npm run start"
+        - "npm run start:wayland"
+        - "npm run start:windows"
+        - "npm run start:x11"
+    validations:
```

Comment:
looks like `npm run server` is missing

Reply:
and start of clientonly is missing too ...

Reply:
I just added both :+1: